source/features/list-prs-for-file.tsx

```diff
@@ -26,7 +26,13 @@ function getDropdown(prs: number[]): HTMLElement {
 					File touched by PRs
 				</div>
 				{prs.map(prNumber => (
-					<li>
+					<li
+						className="issue-link js-issue-link tooltipped tooltipped-ne"
+						data-error-text="Failed to load PR title"
+						data-permission-text="PR title is private"
+						data-url={['', getRepo()?.nameWithOwner, 'issues', prNumber].join('/')}
```

Comment:
we’re definitely in a repo

```suggestion
						data-url={['', getRepo()!.nameWithOwner, 'issues', prNumber].join('/')}
```

Reply:
> We already have a getPRUrl function, look at how it builds the URL internally

Right, I didn't use it because at first I thought the auto-fetching only worked with a relative URL. But actually the problem was `/pull/` in the `data-url` attribute. For some reason GH wants the URL with `/issues/` (which redirects to the pull one anyway) :shrug: